backend/fms_core/viewsets/_constants.py

```diff
@@ -133,6 +133,8 @@
 _instrument_filterset_fields: FiltersetFields = {
     "id": PK_FILTERS,
     "name": CATEGORICAL_FILTERS_LOOSE,
+    "serial_id": CATEGORICAL_FILTERS_LOOSE,
+    "type__id": CATEGORICAL_FILTERS_LOOSE,
```

Comment:
type provides on the model itself the same as type__id ... It would be a FK_FILTERS.